### DISCUSSION TITLE
las-30a-revised-2010.las - how can I get '*_Parameter' and '*_Data' sections for 'Log', 'Drilling', 'Core', 'Inclinometry', 'Test',  'TOPS', 'Perforations' sections?

### DISCUSSION THREAD

**Discussion by volozhanin-gg** [kinverarity1/lasio#488] [Q&A]
I can see them in the file, but cannot find 'Inclinometry_Data' values in   >>>vars(las)
I have only 'Inclinometry_Definition' in   >>> las.section.keys()

How can I get Inclinometry_Parameter and Inclinometry_Data by this key?
If it's not implemented yet, can I count on having it in the next year?

best regards and thanks anyway

**Comment by dcslagel** [comment]
Hi @volozhanin-gg,  Lasio partially reads LAS-3.0 files.  The LAS-3.0 specific metadata header section parsing isn't implemented yet.  The data section should be read successfully.

There is an Issue tracking LAS-3.0 features. So the progress can be checked there: https://github.com/kinverarity1/lasio/issues/5
